krun.py

```diff
@@ -101,109 +103,90 @@ def sanity_check_user_change(platform):
 def create_arg_parser():
     """Create a parser to process command-line options.
     """
-    parser = argparse.ArgumentParser(description='Benchmark, running many fresh processes.')
```

Comment:
Well, up to you, but maybe we should change code as we come to it rather than a big bang. Let me know if you want me to change the strings in this PR.